appinventor/appengine/src/com/google/appinventor/client/wizards/NewProjectWizard.java

```diff
@@ -49,7 +49,8 @@ protected NewProjectWizard(String title) {
    * @param onSuccessCommand command to be executed after process creation
    *                         succeeds (can be {@code null})
    */
-  protected void createNewProject(String projectType, final String projectName,
+  // TODO: This should probably be consolidated in NewYoungAndroidProjectWizard
+  public static void createNewProject(String projectType, final String projectName,
```

Comment:
What is the justification for the change from `protected` to `public static`? The logic for creating the project should be encapsulated within the wizard.